grpcserver/interceptor/logging_interceptor.go

```diff
@@ -135,7 +149,10 @@ func LoggingStreamInterceptor(logger log.FieldLogger, options ...LoggingOption)
 	info *grpc.StreamServerInfo,
 	handler grpc.StreamHandler,
 ) error {
-	opts := &loggingOptions{slowCallThreshold: defaultSlowCallThreshold}
+	opts := &loggingOptions{
+		slowCallThreshold:  defaultSlowCallThreshold,
+		timeSlotsThreshold: defaultTimeSlotsThreshold,
```

Comment:
`timeSlotsThreshold` defaults to the same value as `slowCallThreshold` in the HTTP middleware, but in gRPC it’s `1s`. To make the behavior consistent, I suggest initializing it like this:
```go
opts := &loggingOptions{slowCallThreshold: defaultSlowCallThreshold}
for _, option := range options {
    option(opts)
}
if opts.timeSlotsThreshold == 0 {
    opts.timeSlotsThreshold = opts.slowCallThreshold
}
```
This way, `timeSlotsThreshold` inherits `slowCallThreshold` unless explicitly overridden.